### DISCUSSION TITLE
what does the colors means for the tweet button and the home button ?

### DISCUSSION THREAD

**Discussion by BulgurAkhan** [dimdenGD/OldTwitter#738] [Q&A]
The home button, the bird, is blue and the tweet button on his right also
in some case when I(m on a profile I can see it red, for dimden it is pink
what does this colors mean ? 
thanks

**Comment by dimdenGD** [comment]
Everyone can set their own link color in profile settings

**Reply by BulgurAkhan** [reply]
ok thank you
